frontend/package.json

```diff
@@ -73,7 +73,8 @@
     "typescript": "^4.8.4",
     "universal-cookie": "^4.0.4",
     "uswds": "^2.13.3",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4",
+    "date-fns": "^2.29.3"
```

Comment:
want to go ahead and also update package-lock.json so the CI passes?

Reply:
Gotcha.. It never updated on my end because the whole project requires node <= 17 and thus I've only been building in container. I'll mount it and install there and update the package-lock.